ci/taos/plugins-staging/pr-nnstreamer-ubuntu-apptest.sh

```diff
@@ -126,12 +123,27 @@ function pr-nnstreamer-ubuntu-apptest-run-queue() {
         echo -e "[DEBUG] USB Camera device is not enabled. It is required by {nnstreamer_example_filter|nnstreamer_example_cam}."
         echo -e "[DEBUG] Enabling virtual cam camera..." 
 
+        # Install virtual camera device kernel module, if it is not installed.
```

Comment:
The noun is too long. 

* before: # Install virtual camera device kernel module, if it is not installed.
* after : # Install a 'v4l2loopback' kernel module to use a virtual camera device if it is not installed.